master/docs/manual/cfg-www.rst

```diff
@@ -571,12 +574,9 @@ Here is a configuration that is known to work (Apache 2.4.10 / Debian 8), direct
         ServerName buildbot.example
         ServerAdmin webmaster@buildbot.example
 
-        <Location /ws>
-          ProxyPass ws://127.0.0.1:8020/ws
-          ProxyPassReverse ws://127.0.0.1:8020/ws
-        </Location>
-
-        ProxyPass /ws !
+        # replace with actual port of your Buildbot master
+        ProxyPass ws://127.0.0.1:8020/ws
```

Comment:
@gracinet yes. take a look #3299 (and docs) :D 
Could you post there your full config for bb-master? it is strange for me it works